src/SDK/Api.php

```diff
@@ -82,4 +82,15 @@ public function isMollieOutageException(\Mollie\Api\Exceptions\ApiException $e):
         }
         return false;
     }
+
+    public function isMollieFraudException(\Mollie\Api\Exceptions\ApiException $e): bool
+    {
+        $isFraudCode = $e->getCode() === 422;
+        $isFraudMessage = strpos($e->getMessage(), 'The payment was declined due to suspected fraud') !== false;
```

Comment:
@mmaymo is this the only way to detect fraud? It could be a temporary solution, but they are returning 422 for too many scenarios. This can break any day.

Reply:
You are right, I will raise this next Monday. Thanks!